### ISSUE TITLE
Invalid XML Schema reference

### ISSUE THREAD

**Issue by redmitry** [bio-tools/biotoolsRegistry#404]
Hello,
Trying to get bio.tools data in XML format, 
https://dev.bio.tools/api/tool/?format=xml
got:

```
<tools xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="file:///E:/repos/GitHub/biotoolsShim/genericxml2xml/versions/biotools-3.0.0/biotools_3.0.0.xsd">
```
I think the schema should be located somewhere @ bio.tools or taken from github directly:
https://raw.githubusercontent.com/bio-tools/biotoolsSchema/master/versions/biotools-3.0.0/biotools_3.0.0.xsd

Kind regards,

Dmitry

**Comment by joncison** [comment]
Yup, the schema location needs fixing in the released versions (see https://github.com/bio-tools/biotoolsSchema/issues/78)

Note if you want XML format you should use this endpoint *e.g.*
https://bio.tools/api/tool/signalp?format=xml

which returns properly formatted biotoolsSchema 3.0.0-compliant XML

Do not use the search API, *e.g.*
https://bio.tools/api/tool/?toolid=signalp?format=xml

which returns mangled XML currently (see https://github.com/bio-tools/biotoolsRegistry/issues/403)

**Comment by hansioan** [comment]
@redmitry @joncison 
Both dev.bio.tools and bio.tools should work the same, but the content of dev is more like a sandbox.
